settings/Database.boot.setting.php

```diff
@@ -0,0 +1,124 @@
+<?php
+/*
+ +--------------------------------------------------------------------+
+ | Copyright CiviCRM LLC. All rights reserved.                        |
+ |                                                                    |
+ | This work is published under the GNU AGPLv3 license with some      |
+ | permitted exceptions and without any warranty. For full license    |
+ | and copyright information, see https://civicrm.org/licensing       |
+ +--------------------------------------------------------------------+
+ */
+
+/**
+ *
+ * @package CRM
+ * @copyright CiviCRM LLC https://civicrm.org/licensing
+ */
+
+/**
+ * Settings metadata file
+ */
+return [
+  'civicrm_db_dsn' => [
+    'name' => 'civicrm_db_dsn',
+    'title' => ts('CiviCRM Database Connection String'),
+    'description' => ts("The DSN for the CiviCRM Database."),
+    'group' => 'database',
+    'group_name' => 'CiviCRM Database',
+    'type' => 'String',
+    'html_type' => 'text',
+    //'default' => '',
+    'add' => '5.76',
+    'is_domain' => '1',
+    'is_contact' => 0,
+    'help_text' => NULL,
+    'is_constant' => TRUE,
+    'load_from_env' => TRUE,
+    'fqn' => 'CIVICRM_DSN',
+  ],
+  'civicrm_db_name' => [
+    'name' => 'civicrm_db_name',
+    'title' => ts('CiviCRM Database Name'),
+    'description' => ts("The database name component of the DSN for the CiviCRM Database."),
+    'group' => 'database',
+    'group_name' => 'CiviCRM Database',
+    'type' => 'String',
+    'html_type' => 'text',
+    'default' => 'civicrm',
+    'add' => '5.76',
+    'is_domain' => '1',
+    'is_contact' => 0,
+    'help_text' => NULL,
+    'is_constant' => TRUE,
+    'load_from_env' => TRUE,
+    'fqn' => 'CIVICRM_DB_NAME',
```

Comment:
'fully_qualified_name' ? something else?

(used for php constant names and env variable names)

Reply:
Hopefully we can avoid bikeshedding it too much, but we tend to stay away from acronyms and abbreviations due to some bad experiences in the past (ahem, `civicrm_uf_...`). Maybe just `full_name` if that's clear?

Reply:
I was reaching for something with a bit more indication of how/why different from 'name'... but I also dislike abbreviations. `global_name` ?

Reply:
`global_name` sounds fine